docs/DeveloperGuide.md

```diff
@@ -326,29 +326,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user preparing interviews | be able to view all candidates who will be interviewed on a particular date | keep track of the interviews planned for that day |
-| ` * * `  | user preparing interviews | be able to delete the interview sessions on a particular date | |
-| `  *  `  | user preparing interviews | add a co-interviewer to an interview | I know who I will be interviewing the candidate with |
-| ` * * `  | user after interviews | be able to mark a candidate as ‘Interviewed’ | |
 | `* * *`  | user after interviews | be able to tag a candidate e.g. by the position they are applying for | group them according to the tags |
-| `* * *`  | user       | want to be able to edit existing tags of a candidate | |
-| `* * *`  | user       | be able to remove existing tags of a candidate | |
+| ` * * `  | user       | see all the candidates scheduled for interview sessions for a particular job posting | |
 | ` * * `  | user searching for candidates | be able to filter the candidates for some specific requirements | find the right person for the post more efficiently |
-| `* * *`  | user searching for candidates | be able to search for candidates who have a certain remark/description | find candidates by criteria |
 | ` * * `  | user searching for candidates |  be able to search who applied for certain positions |  I know all the candidates for that specific position |
+| ` * * `  | user after interviews | be able to mark a candidate as ‘Interviewed’ | |
+| ` * * `  | user preparing interviews | be able to schedule a new interview for a candidate | recruit more talent |
+| ` * * `  | user preparing interviews | be able to delete all other scheduled interview sessions for a particular candidate once I decide to assign him a particular post | |
+| ` * * `  | user preparing interviews | be reminded of what interviews I have the next day | keep track of them |
+| ` * * `  | user preparing interviews | be able to reschedule the interview session for a candidate | so that I can keep up with changes coming from the candidate's end |
+| ` * * `  | user preparing interviews | be able to delete the interview sessions on a particular date | |
 | ` * * `  | user preparing interviews | search what interviews I have for a particular date | better prepare for that day |
-| ` * * `  | user       | see all the candidates scheduled for interview sessions for a particular job posting | |
-| `* * *`  | user       | be able to search for a particular candidate to see his/her upcoming sessions | |
+| `  *  `  | user preparing interviews | add a co-interviewer to an interview | I know who I will be interviewing the candidate with |
 | `  *  `  | user       | be able to password lock the application to prevent unauthorised access | |
 | `  *  `  | user       | encrypt the save file | prevent my data from being easily stolen |
```

Comment:
I think the spaces between asterisk and ` need to be removed too for the DG webpage to display correctly

Reply:
resolved